src/manifest_fenix.json

```diff
@@ -0,0 +1,70 @@
+{
+  "manifest_version": 2,
```

Comment:
Do we need to bump the manifest_version on changes?

Reply:
:)